Cargo.toml

```diff
@@ -71,7 +69,7 @@ syn = { version = "1.0" }
 hex = { version = "0.4.3", default-features = false }
 paste = "1.0.6"
 slices = "0.2.0"
-schnorrkel = { verison = "11.4", default-features = false }
+schnorrkel = { version = "0.9.1", default-features = false }
```

Comment:
Ah its cause I spelled it wrong and forgot 0, we should use the latest version: https://crates.io/crates/schnorrkel

Reply:
The latest version does not compile, errors for the curve2559 crate.